examples/Esp32-mqtt/Esp32-mqtt.ino

```diff
@@ -13,77 +13,23 @@
  * limitations under the License.
  *****************************************************************************/
 #include <Arduino.h>
+#include <CloudIoTCore.h>
+#include <CloudIoTCoreMQTTClient.h>
 #include <WiFi.h>
-#include <WiFiClientSecure.h>
+#include <ciotc_config.h>  // Configure with your settings
```

Comment:
Did you mean:

    "ciotc_config.h"

My path doesn't search local without the quotes. Otherwise LGTM!